core/kernel/uri/Bin2HexUriProvider.php

```diff
@@ -49,7 +49,7 @@ class Bin2HexUriProvider extends ConfigurableService implements UriProvider
      */
     public function provide()
     {
-        return $this->getOption(self::OPTION_NAMESPACE) . uniqid('i') . getmypid()
-            . bin2hex(openssl_random_pseudo_bytes(8));
+        return $this->getOption(self::OPTION_NAMESPACE) . uniqid('i') . date('YmdHis')
```

Comment:
Thanks!

- [ ] Please confirm that this is the strategy BOSA uses on PROD.
- [ ] Please add unit tests (at least to check that it always has 36 length).

Reply:
Done 🙂